### PR TITLE
#203 Return to MyTripFragment from AddNewTripActivity

### LINKED ISSUE #203: Doesn't redirect back on adding a trip
Describe your issue here.

### Steps to reproduce

My Trips -> Add new Trip

### Expected behaviour
Application should redirect back on the list of the Trips 

### Actual behaviour
Display a Toast of successful addition of trip and remain on same activity

### Would you like to work on the issue?
yes

### PR DESCRIPTION
**Pull request by f71uday**
# Description

added startActivityForResult 

Fixes #203 

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by f71uday** [comment]
What should be the styling?

**Comment by f71uday** [comment]
@Swati4star please help me with this, 
this time I have resolved the issue

**Comment by f71uday** [comment]
@Swati4star there are no errors in the functionality, and I have no idea how to deal with checkstyle errors if there are any references where I can learn things, please suggest

**Comment by Swati4star** [comment]
Check the travis link .Every error is mentioned in that link

**Comment by Swati4star** [comment]
@f71uday I don't think you have tested it. I just pulled the changes locally & this isn't working.

**Comment by f71uday** [comment]
@Swati4star no i have performed all functional test . its working fine in my local

**Comment by Swati4star** [comment]
I clciked on + icon. Filled the details. Clicked on `Add Trip`. It went back to previous page & toast was displayed `Trip added`.
But I couldn't see the newly added trip in the list.
You can look at this video also: https://photos.app.goo.gl/2uAz3xaBEkMU2cd9A

File: Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsFragment.java [hunk 1]
```diff
@@ -166,4 +168,14 @@ public void onAttach(Context activity) {
         super.onAttach(activity);
         this.mActivity = (Activity) activity;
     }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+        if (requestCode == ADDNEWTRIP_ACTIVITY) {
+            Toast.makeText(getContext(),"Activity callback",Toast.LENGTH_LONG).show();
```
[the remark this answers is not in the document]
Comment (f71uday) [reply Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsFragment.java]: Done

**Comment by f71uday** [comment]
All thanks to you @Swati4star  finally I made my first open source contribution.

**Comment by Swati4star** [comment]
@f71uday This is awesome! You can pick up any other issue also if you want.
You must have got familiar with the project now :D

**Pull request merged**